pkgs/applications/misc/remarkable/rmview/default.nix

```diff
@@ -18,6 +18,10 @@ python3Packages.buildPythonApplication rec {
     pyrcc5 -o src/rmview/resources.py resources.qrc
   '';
 
+  preFixup = ''
+    wrapQtApp "$out/bin/rmview"
```

Comment:
This wraps the already existent python wrapper and multiple wrapping should be avoided.
You should be able to add `qtWrapperArgs` to what the python wrapper uses, or vice versa (I have no experience with python nixpkgs).